pages/deploy/deploy_aws.md

```diff
@@ -558,21 +575,22 @@ In the example config.json, the feature that enables file upload from data.all U
+| file_actions                            | s3_datasets | Create, Read, Update, Delete on Dataset Folders                                                                                                                                                                                                                                              |
+| aws_actions                             | s3_datasets | Get AWS Credentials and assume Dataset IAM role from data.all's UI                                                                                                                                                                                                                           |
+| preview_data                            | s3_datasets | Enable previews of dataset tables for users in data.all UI                                                                                                                                                                                                                                   |
+| glue_crawler                            | s3_datasets | Allow running Glue Crawler to catalog new data for data.all datasets directly from the UI                                                                                                                                                                                                    |
+| share_notifications                     | s3_datasets | Allow additional notifications (on top of data.all's built in UI notifications) to be sent to data.all users when a dataset sharing operation occurs (currently only type `email` notifications is supported and requires `custom_domain` hosted zone parameters be specified in `cdk.json`) |
+| confidentiality_dropdown                | s3_datasets | Disable / Enable use of confidentiality levels for a dataset. Please note - when this drop down is set to false each dataset is treated as if it is Official or Secret                                                                                                                       |
+| topics_dropdown                         | s3_datasets | Disable / Enable use of topics for a dataset                                                                                                                                                                                                                                                 | 
+| auto_approval_for_confidentiality_level | s3_datasets | Specify if auto-approval for share requests should be enabled for each confidentiality level in data.all                                                                                                                                                                                     |
```

Comment:
It would be good to explain which value does what because it is not a simple enable/disable.

Reply:
+ we are missing share_logs